### DISCUSSION TITLE
Using skywalking as APM in kubernetes cluster

### DISCUSSION THREAD

**Discussion by madhavichinni** [apache/skywalking#9413] [Q&A]
I am exploring Skywalking to use it for JVM Profiling use cases like capturing Heap Dumps, Memory usage per call stack, Thread dumps, Garbage Collection and Web Requests profiling like request count, CPU %. We want to use Skywalking JVM profiling for the applications are deployed in Kubernetes cluster and EC2.
Could you share some links which have the instructions on how to user Skywalking for applications deployed in Kubernetes cluster and EC2.

**Comment by kezhenxu94** [comment]
Check this repo https://github.com/apache/skywalking-showcase

**Reply by wu-sheng** [reply]
There are many ways
- Helm, https://skywalking.apache.org/docs/main/v9.1.0/en/setup/backend/backend-k8s/
- Operator, https://skywalking.apache.org/docs/skywalking-swck/v0.6.1/readme/

Although, the showcase is the best quick start.
